octoversion.json

```diff
@@ -1,3 +1,3 @@
 {
-    "NonPreReleaseTags": ["refs/heads/main"]
+    "NonPreReleaseTags": ["refs/heads/main", "main"]
```

Comment:
```suggestion
    "NonPreReleaseTags": ["main"]
```

We shouldn't need that once we are only building here in Actions.

Reply:
Lols. I just asked Edlyn to add it :)